README.md

```diff
@@ -0,0 +1,22 @@
+# Awesome Book Store Project
+This is a project that I and my coding partner have created using HTML,CSS and js. For now it have the functionality of storing the book: name and author and you can removed also the specific book. The data will be stored in user browser.
+
+Built With
+HTML
+CSS
+JS
+Author
+👤 Author1
+
+GitHub: @Hidar37
+👤 Author2
+
+GitHub: @c00p75
+
+Show your support
+Give a ⭐️ if you like this project!
+
+📝 License
+This project is MIT licensed.
+
+NOTE: we recommend using the MIT license - you can set it up quickly by using templates available on GitHub. You can also use any other license if you wish.
```

Comment:
- [ ] Kindly adhere to the suggestion by the previous reviewer. Kindly use this [new template](https://github.com/microverseinc/readme-template) to update your README file.

Reply:
- Please use the new [readme](https://github.com/microverseinc/readme-template) template suggested by the previous reviewers for your project to get reviewed/ approved 🙏 🙏